edx_prefectutils/mysql.py

```diff
@@ -3,7 +3,7 @@
 """
 import os
 
-import mysql.connector
```

Comment:
Curious about needing to change this ?

Reply:
@HassanJaveed84 so mysql.connector had a lot of dependency issues with prefect new version and it required pinning a lot of things so I thought rather doing that using a other mysql connector don't decency issue will be a better option.